kokoro/linux/dockerfile/test/php/Dockerfile

```diff
@@ -232,6 +235,48 @@ RUN wget -O phpunit https://phar.phpunit.de/phpunit-7.phar \
   && cp phpunit /usr/local/php-7.3/bin \
   && mv phpunit /usr/local/php-7.3-zts/bin
 
+# php 7.4
+RUN wget http://ftp.gnu.org/gnu/bison/bison-3.0.1.tar.gz -O /var/local/bison-3.0.1.tar.gz
```

Comment:
Could you make this an https link just to be safe?

Reply:
Done

Reply:
done